dev/com.ibm.ws.microprofile.faulttolerance.1.1/resources/META-INF/services/com.ibm.ws.microprofile.faulttolerance.spi.FaultToleranceProviderResolver

```diff
@@ -0,0 +1 @@
+com.ibm.ws.microprofile.faulttolerance.impl.ProviderResolverImpl
```

Comment:
i think in config we decided that this file is not needed and can cause problems with the way we have the projects set up. it isn't needed because the provider resolver is set programatically when the DS starts